scripts/export-meshseg-onnx.py

```
@@ -468,9 +482,17 @@ def blob(c, r, label, w, squash=None, solid=False):
-            parts.append((lambda n, a=base, b=tip, r=trunkR * rng.uniform(0.3, 0.6):
-                          capsule_surf(a, b, r, n, rng), BRANCH, bl * trunkR))
+            br = trunkR * rng.uniform(0.35, 0.65)
+            parts.append((lambda n, a=base, b=tip, r=br:
+                          capsule_surf(a, b, r, n, rng), BRANCH, bl * trunkR * 3))
             tips.append(tip)
+            # sub-branches continuing from the tip deeper into the canopy
+            for _ in range(rng.integers(0, 3)):
+                d = _unit_dirs(1, rng)[0]; d[1] = abs(d[1]) * rng.uniform(0.3, 1.0)
+                sl = bl * rng.uniform(0.3, 0.7)
+                stip = tip + d * sl
+                parts.append((lambda n, a=tip, b=stip, r=br * rng.uniform(0.5, 0.8):
```

Ruff (0.15.21)

[warning] Do not perform function call `rng.uniform` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)